chart/compass/values.yaml

```diff
@@ -23,7 +23,7 @@ global:
       version: "PR-1460"
     director:
       dir:
-      version: "PR-1504"
+      version: "PR-1529"
```

Comment:
pairing is also possible with client credentials (for apps, int systems and runtimes) - i didnt see new logs there